hack/update-estimator-protobuf.sh

```diff
@@ -14,7 +14,7 @@ export PATH=$PATH:$GOPATH/bin
 GO111MODULE=on go install golang.org/x/tools/cmd/goimports
 GO111MODULE=on go install k8s.io/code-generator/cmd/go-to-protobuf
 GO111MODULE=on go install github.com/gogo/protobuf/protoc-gen-gogo
-GO111MODULE=on go install github.com/vektra/mockery/v2
+GO111MODULE=on go install github.com/vektra/mockery/v2@v2.10.0
```

Comment:
If it is due to the version.
I guess we should update the version at the [go.mod](https://github.com/karmada-io/karmada/blob/86649395b95a08f1b350281ec105a0aa9f7a89f0/go.mod#L17). Not here.

Reply:
That's right, thanks for correction.